lib/ruby_indexer/test/index_test.rb

```diff
@@ -1822,5 +1822,161 @@ class Child < Namespace::Parent
         @index.linearized_ancestors_of("Foo::Child::<Class:Child>"),
       )
     end
+
+    def test_augmenting_indexing_included_hook
```

Comment:
How about we introduce a seperate tes file for things related to the augmentations?

Reply:
Good idea.